res/layout/share_button_wrapper.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ViewSwitcher xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Using ViewSwitcher is very nice here. Would it be possible to use a ViewFlipper instead, to separate the post button and login button ?
http://developer.android.com/reference/android/widget/ViewFlipper.html